src/usr/local/www/system_advanced_network.php

```diff
@@ -158,23 +158,25 @@
 ))->setHelp('NOTE: This does not disable any IPv6 features on the firewall, it only '.
 	'blocks traffic.');
 
+
 $group = new Form_Group('IPv6 over IPv4 Tunneling');
 $group->add(new Form_Checkbox(
 	'ipv6nat_enable',
 	'IPv6 over IPv4 Tunneling',
-	'Enable IPv4 NAT encapsulation of IPv6 packets',
+	'Enable IPv6 over IPv4 tunneling',
```

Comment:
It's going to repeat either way with both wordings - either repeating the section label or repeats the help text that immediately follows. My preference is based on the thought that "nat encapsulation" is a more technical term some users will know and some won't (and is in the help text already for those needing it), whereas "tunneling" is immediately obvious to everyone and therefore more easy to recognise if the user is looking for it.

Maybe just "Enable IPv6 over IPv4" rather than either, and trust to the help text and section label to explain the rest of it?